modules/nf-scil/betcrop/synthbet/tests/nextflow.config

```diff
@@ -0,0 +1,6 @@
+process {
+    withName: "BETCROP_SYNTHBET" {
+        publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+        ext.nocsf = true
+    }
```

Comment:
Can you test if you had this to your config file if the test are still passing? 
`memory = 8.GB`
This would change the allocated memory only for your module, and not the whole library.

Reply:
Passed!